service/sg_accel_service_install.sh

```diff
@@ -4,18 +4,19 @@
 OS=""
 VER=""
 SERVICE_NAME="sg_accel"
-SRCCFGDIR=../examples
+SCRIPT_DIR="$( cd "$( dirname "$0" )" > /dev/null && pwd )"
```

Comment:
I'm having trouble understanding what this does .. it sounds like it's:

* Figuring out the dir of this script itself (dirname $0)
* Then cd'ing into it?  If that's true is confusing that it would cd while setting a variable
* I'm confused about ` > /dev/null && pwd`

Can you add some comments which explains what it does?  And then maybe simplify it if possible..